LiskHQ/lisk-sdk
# Fix core crash - Closes #7368, #7447 and #7591

### What was the problem?

This PR resolves #7368 , resolves #7447 and resolves #7591

1. In case of application success to commit but engine fails, it leaves the data at inconsistent state
2. If genesis state execution takes too long, it leaves the data at inconsistent state
3. ABI request was not counting the ID, so if there are multiple calls at the same time, req/res will be mixed
4. `oclif-dev` command was outdated

### How was it solved?

- Update ABI call from `ready` to `init` and add state revert logic in case of inconsistent state
- Update `oclif-dev` to use `oclif` command
- Update ABI client to count the ID

### How was it tested?

- Run lisk-core and lisk-service together
or
- Run dpos-mainchain and make multiple RPC request and see it doesnt crash
